src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig

```diff
@@ -110,41 +110,50 @@
     margin-top: 30px;
 }
 .modal-content .settings-group {
+    border: 1px solid var(--settings-option-border-color);
+    border-radius: 4px;
     display: flex;
     margin-bottom: 15px;
 }
-.modal-content label {
+.modal-content .settings-group label {
     cursor: pointer;
     display: flex;
     flex: 1;
     font-size: 16px;
     margin: 0;
 }
-.modal-content label input {
+.modal-content .settings-group label input {
     display: none;
```

Comment:
instead of `display:none`, we should probably hide them only visually to keep them focusable with keyboard (and adding styles for `input:focus-visible + p` to make that focus visible).

Styles to hide them visually:

```css
.modal-content .settings-group label input {
  position: absolute;
  clip: rect(0, 0, 0, 0);
  pointer-events: none;
  opacity: 0;
}
```